test/test-with-3-plugins.sh

```diff
@@ -9,7 +9,8 @@ ADDITIONAL_PLUGIN_1=$4
 ADDITIONAL_PLUGIN_2=$5
 ADDITIONAL_PLUGIN_3=$6
 
-bash ./test/platform-add.sh $CORDOVA_VERSION $PLATFORM $PLATFORM_VERSION
```

Comment:
Is this what made the build pass? If yes, I'll submit a separate PR with this fix, so all the broken PRs would start passing after rebasing.

Reply:
Not really, all but yes, it did fixed a few test cases. Since the cordova & platform version passed to this script from the parent script are same hence there was no point reinstalling it again so I commented out that. But I would love to see the detailed error when this script was trying to install the same cordova & platform.

Reply:
I would also love to help you on rebasing PRs & getting them merged which are good to go because I saw various good PRs

Reply:
how can we go forward ?

Reply:
@felipeclopes can tell anything

Reply:
I'm not really sure hot to proceed. It seems that the first step would be to try to fix the test cases that are broken. If you look at the list of PRs to merge, the only ones passing are the ones who removed the tests for older Cordova versions and this one. 

Maybe a separate PR just with this fix of the build cold help the other PRs to rebase and get a green status on the build system.

Reply:
In my thinking, we can start with merging this branch to master then start rebasing other PR with master so most of the PRs will by default pass

Reply:
@felipeclopes any update? I'll still suggest to move forward with this